lib/src/test/java/com/thenewboston/utils/BankApiMockEngine.kt

```diff
@@ -0,0 +1,88 @@
+package com.thenewboston.utils
+
+import io.ktor.client.*
+import io.ktor.client.engine.mock.*
+import io.ktor.client.features.json.*
+import io.ktor.client.features.json.serializer.*
+import io.ktor.http.*
+
+class BankApiMockEngine {
+
+
+
+    fun getSuccess() = getBankMockEngine()
+
+    fun getErrors() = getBankMockEngine(true)
+
+    private val responseHeaders = headersOf("Content-Type" to listOf(ContentType.Application.Json.toString()))
+
+    private fun getBankMockEngine(enableErrorResponse: Boolean = false) = HttpClient(MockEngine) {
+        engine {
+            addHandler { request ->
+                when (request.url.encodedPath) {
+                    BankAPIJsonMapper.ACCOUNTS_ENDPOINT -> {
+                        if (enableErrorResponse) {
+                            respond(BankAPIJsonMapper.mapAccountsToJson(), HttpStatusCode.InternalServerError, responseHeaders)
```

Comment:
Is there a reason we're still getting success response bodies in the error cases? Not sure how exactly the services behave but I'd expect an empty or some error response.

Reply:
Yeah I've asked Bucky what the error response bodies and they are currently working on the documentation roadmap. https://github.com/thenewboston-developers/Management/issues/29

I guess for now, we can just create a fake dummy error response and change it later.